### ISSUE TITLE
Setting static property to argument

### ISSUE THREAD

**Issue by Jurigag** [zephir-lang/zephir#1446]
https://github.com/phalcon/zephir/pull/1445/files check this pr for more, when trying to set static argument to non-static method argument then the value is just `*uninitialized*`

**Comment by Jurigag** [comment]
Well it passed somehow on travis, need to check what's going on

**Comment by Jurigag** [comment]
It might be parser issue though @sjinks @sergeyklay not sure about this

**Comment by sjinks** [comment]
Yes, that's compiler's issue:

```
zephir/ext/test/spropertyaccess.zep.c:94:34: warning: passing argument 1 of ‘zephir_read_static_property_ce’ from incompatible pointer type [-Wincompatible-pointer-types]
   zephir_read_static_property_ce(&delimiter, test_spropertyaccess_ce, SL("_delimiterWithUnderscore"), PH_NOISY_CC);
```

plus possible memory corruption

**Comment by sjinks** [comment]
Fixed in 6c6f7bf0d145f3a3091acc32606fe2857e98ff4b
